docs/pages/server-access/guides/vscode.mdx

```diff
@@ -111,16 +129,15 @@ Host node000.foo.example.com
 When finished, save the file. If the added host doesn't automatically appear in
 the list, select the Refresh button.
 
-## Step 3/3. Start a Remote Development Session
+## Step 3/3. Start a Remote Development session
 
```

Comment:
I don't understand the capitalization here.

Is it a remote _development session_, or a _remote development_ session. 🤷‍♂️

Reply:
It looks like Remote Development is a product name, so I capitalized that (https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.vscode-remote-extensionpack). If that gets in the way of clarity, though, I could ignore the product name and just say "Start a remote development session."